tools/spdx-check/spdx-check.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+BASE_COMMIT=$1
+SCRIPT_DIR=$(dirname "$0")
+IGNORE_FILE="$SCRIPT_DIR/.spdxignore"
```

Comment:
I would consider the file from the repo root dir, i.e., where the script is called, we have other files like this as well: 

```
.codespellignorelines
.codespellrc
.dockerignore
.gitignore
.hadolint.yaml
.markdownlint.yaml
.readthedocs.yaml
.revive.toml
.syft.yaml
.yamllint
.yetus-excludes
```

Reply:
Yeah... Maybe it will make usage easier. I'll redo now

Reply:
Done